client/asset/ltc/spv.go

```diff
@@ -0,0 +1,1046 @@
+	if err != nil {
+		return nil, err
+	}
+	return btcutil.DecodeAddress(ltcAddr.String(), w.btcParams)
+}
+
+func (w *ltcSPVWallet) NewAddress(account uint32, _ waddrmgr.KeyScope) (btcutil.Address, error) {
+	ltcAddr, err := w.Wallet.NewAddress(account, ltcwaddrmgr.KeyScopeBIP0084)
+	if err != nil {
+		return nil, err
+	}
+	return btcutil.DecodeAddress(ltcAddr.String(), w.btcParams)
+}
+
+func (w *ltcSPVWallet) PrivKeyForAddress(a btcutil.Address) (*btcec.PrivateKey, error) {
+	ltcAddr, err := ltcutil.DecodeAddress(a.String(), w.chainParams)
+	if err != nil {
+		return nil, err
+	}
+
+	ltcKey, err := w.Wallet.PrivKeyForAddress(ltcAddr)
+	if err != nil {
+		return nil, err
+	}
+
+	priv, _ /* pub */ := btcec.PrivKeyFromBytes(ltcKey.Serialize())
+	return priv, nil
+}
+
+func (w *ltcSPVWallet) SendOutputs(outputs []*wire.TxOut, _ *waddrmgr.KeyScope, account uint32, minconf int32,
+	satPerKb btcutil.Amount, css btcwallet.CoinSelectionStrategy, label string) (*wire.MsgTx, error) {
+
+	ltcOuts := make([]*ltcwire.TxOut, len(outputs))
+	for i, op := range outputs {
+		ltcOuts[i] = &ltcwire.TxOut{
+			Value:    op.Value,
+			PkScript: op.PkScript,
+		}
+	}
+
+	ltcTx, err := w.Wallet.SendOutputs(ltcOuts, &ltcwaddrmgr.KeyScopeBIP0084, account,
+		minconf, ltcutil.Amount(satPerKb), wallet.CoinSelectionStrategy(css), label)
+	if err != nil {
+		return nil, err
+	}
+
+	btcTx, err := convertMsgTxToBTC(ltcTx)
+	if err != nil {
+		return nil, err
+	}
+
+	return btcTx, nil
+}
+
+func (w *ltcSPVWallet) HaveAddress(a btcutil.Address) (bool, error) {
+	ltcAddr, err := ltcutil.DecodeAddress(a.String(), w.chainParams)
+	if err != nil {
+		return false, err
+	}
+
+	return w.Wallet.HaveAddress(ltcAddr)
+}
+
+func (w *ltcSPVWallet) Stop() {
+	w.log.Info("Unloading wallet")
+	if err := w.loader.UnloadWallet(); err != nil {
+		w.log.Errorf("UnloadWallet error: %v", err)
+	}
+	if w.chainClient != nil {
+		w.log.Trace("Stopping neutrino client chain interface")
+		w.chainClient.Stop()
+		w.chainClient.WaitForShutdown()
+	}
+	w.log.Trace("Stopping neutrino chain sync service")
+	if err := w.cl.Stop(); err != nil {
+		w.log.Errorf("error stopping neutrino chain service: %v", err)
+	}
+	w.log.Trace("Stopping neutrino DB.")
+	if err := w.neutrinoDB.Close(); err != nil {
+		w.log.Errorf("wallet db close error: %v", err)
+	}
+
+	w.log.Info("SPV wallet closed")
+}
+
+func (w *ltcSPVWallet) AccountProperties(_ waddrmgr.KeyScope, acct uint32) (*waddrmgr.AccountProperties, error) {
+	props, err := w.Wallet.AccountProperties(ltcwaddrmgr.KeyScopeBIP0084, acct)
+	if err != nil {
+		return nil, err
+	}
+	return &waddrmgr.AccountProperties{
+		AccountNumber:        props.AccountNumber,
+		AccountName:          props.AccountName,
+		ExternalKeyCount:     props.ExternalKeyCount,
+		InternalKeyCount:     props.InternalKeyCount,
+		ImportedKeyCount:     props.ImportedKeyCount,
+		MasterKeyFingerprint: props.MasterKeyFingerprint,
+		KeyScope:             waddrmgr.KeyScopeBIP0044,
```

Comment:
84?
Not that `numDerivedAddresses()` needs the `KeyScope` value set at all, but it should probably reflect the actual keyscope requested above this.